docs/development/project-development-environment.md

```diff
@@ -133,7 +133,7 @@ ip route get 1 | awk '{print $NF;exit}'
 
 You can generate the complete URL in `sh` with:
 ```bash
-echo "http://$(ip route get 1 | awk '{print $NF;exit}'):8888"
+echo "http://$(ip route get 1 | awk '{print $NF;exit}'):8889"
```

Comment:
Müsste das `awk`-Kommando hier auch angepasst werden?

Reply:
Yes, missed that one, thanks.